assemble/bin/start-daemon.sh

```diff
@@ -103,6 +103,9 @@ if [[ "$SERVICE" != "tserver" || $NUM_TSERVERS -eq 1 ]]; then
    fi
    echo "Starting $SERVICE on $HOST"
 
+   OUTFILE="${ACCUMULO_LOG_DIR}/${SERVICE}_1_${LOGHOST}.out"
```

Comment:
I think we can just bring the entire OUTFILE/ERRFILE definition and `rotate_log` call into the if-branch like you did for the else-branch, and remove the original declaration.

Although, I wonder why we even need to two branches? Didn't this problem come about because we weren't using the same logic? If we hit the "start multiple tservers" branch and only start 1 tserver, would it do what we want?